### PR TITLE
Added TensorFlow example

### PR DESCRIPTION
**Pull request by prodjito**
Adding a TensorFlow model instrumented with the New Relic Python Telemetry SDK. I got a lower RMSE for this model which may indicate it's predicting better on the test set compared to the XGBoost model.

### CODE REVIEW

File: examples/TensorFlow_on_California_housing_prices_dataset.ipynb [hunk 1]
```diff
@@ -0,0 +1,1309 @@
...
+      "execution_count": 3,
+      "id": "e984c34f",
+      "metadata": {
+        "id": "e984c34f",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "62febb56-0560-480b-8b48-1730ca48270d"
+      },
+      "outputs": [
```
Comment (bonczj) [inline examples/TensorFlow_on_California_housing_prices_dataset.ipynb:51]: Is there any reason to keep the output from your tests in the file? Or should the outputs be cleared before merging? I did not check how many there were, but there are a number of output streams included in the file.

Comment (prodjito) [reply examples/TensorFlow_on_California_housing_prices_dataset.ipynb]: I saw that the XGBoost example includes some output so I thought I should include my output as well. But I have now cleared the output, pushed a new commit and squashed the commits. There's no output now.

Comment (bonczj) [reply examples/TensorFlow_on_California_housing_prices_dataset.ipynb]: Thank you! Yes, it is a bit inconsistent, but we'll get that sorted out.

**Review by bonczj — APPROVED** [review]

File: examples/TensorFlow_on_California_housing_prices_dataset.ipynb (hunk 1 above)
Comment (prodjito) [reply examples/TensorFlow_on_California_housing_prices_dataset.ipynb]: Thanks!
